src/doc/ESMF_quickstart.tex

```diff
@@ -461,3 +461,46 @@ \subsubsection{Building and using bundled ESMF Command Line Tools}
 instructions of the individual tools are available in the "Command Line Tools"
 section of the {\it ESMF Reference Manual}.
 
+\subsection{Building ESMF with Spack}
+
+In addition to the manual installation, ESMF can be installed through use of \htmladdnormallink{{\it Spack}}{https://spack.io} package manager. Since there are several ways to install Spack packages, this section aims to demostrate creating new isolated Spack environment to install ESMF library.
```

Comment:
```suggestion
In addition to the manual installation, ESMF can be installed through use of the \htmladdnormallink{{\it Spack}}{https://spack.io} package manager. Since there are several ways to install Spack packages, this section aims to demonstrate creating a new, isolated Spack environment to install the ESMF library.
```

Reply:
done